src/Reflection/ReflectionFunction.php

```diff
@@ -114,4 +116,77 @@ public function getExtensionName() : ?string
     {
         return null;
     }
+
+    /**
+     * @return Closure
+     *
+     * @throws NotImplemented
+     * @throws FunctionDoesNotExist
+     */
+    public function getClosure() : Closure
+    {
+        $this->assertIsNoClosure();
+
+        $functionName = $this->getName();
+
+        $this->assertFunctionExist($functionName);
+
+        return function (...$args) use ($functionName) {
+            return $functionName(...$args);
+        };
+    }
+
+    /**
+     * @param mixed ...$args
+     *
+     * @return mixed
+     *
+     * @throws NotImplemented
+     * @throws FunctionDoesNotExist
+     */
+    public function invoke(...$args)
+    {
+        return $this->invokeArgs($args);
+    }
+
+    /**
+     * @param mixed[] $args
+     *
+     * @return mixed
+     *
+     * @throws NotImplemented
+     * @throws FunctionDoesNotExist
+     */
+    public function invokeArgs(array $args = [])
+    {
+        $this->assertIsNoClosure();
```

Comment:
Why can't you call it on a function?

Reply:
I think that the only possible solution for `invokeArgs` working for closures is to build PHP code from AST and use `eval`. And it will fail anyway for code like this:

```php
<?php

class Foo
{
    private $foo = 'foo';
    
    public function boo()
    {
        return (function () {
            return $this->foo;
        })();
    }
}

Reply:
Ah, got it. Didn't consider that the closure wasn't loaded yet.